src/app/modules/datagrid/datagrid-controller.js

```diff
@@ -317,7 +323,7 @@
 					}
 				],
 				enableFiltering: true,
-				rowHeight: 40,
+				rowHeight: 50,
```

Comment:
let's add a comment (or a named const) to remind us that we have this too-big-for-one--row-but-two-small-for-two row height in order to let the user see that there is more to see if they click.